lib/board.rb

```diff
@@ -0,0 +1,34 @@
+class Board
+  attr_accessor :arr
```

Comment:
- [ ] Use attr_readers/attr_writers instead of attr_accessors. This is to prevent the variable from being modified outside the class which weakens the security of your methods. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))